app/views/admin/failed_bikes/index.html.haml

```diff
@@ -1,16 +1,13 @@
-
 %h1
   Manage Failed Bikes
 
 %h4
-  = @b_params.count
+  = number_with_delimiter(@b_params_total_count, delimiter: ',')
```

Comment:
You can get this with `@b_params.total_count`, I think?

Reply:
ah good catch thanks 👍